doc/source/user/sql_sqlite_dialect.rst

```diff
@@ -115,6 +116,12 @@ between double quotes, the internal double quotes must be escaped with \\
 
     ogrinfo p.shp -sql "SELECT geometry \"FROM\", \"3D\" FROM p"
 
+If single quotes are available in the shell, one can also use just
+
```

Comment:
Single quotes are available in the Windows shell but they can be used only on Linux (and Mac?). I am not sure if it is good to recommend using a syntax that does not work on all platforms.

There needs to be a comma after "geometry".